app/grandchallenge/evaluation/templates/evaluation/partials/phase_admin_warnings.html

```diff
@@ -22,9 +22,9 @@
     </div>
 {% endif %}
 
-{% if phase.parent %}
-    <div class="alert alert-danger" role="alert">
-        <i class="fa fa-exclamation-triangle" aria-hidden="true"></i>
+{% if phase.parent and not hide_phase_parent_info %}
```

Comment:
Why not have it as `{% if phase.parent and show_phase_parent_info %}` to avoid the double negatives?

Reply:
The double negative allowed it to be shown by default since non-existing context variables are false. IIRC the partial template is in use by several (5-7?) other templates. Between the double negative versus repeating a `show_phase_parent_info` many times, I opted for the first.

However, I just found the filter `default_if_none` which makes for a slightly cleaner approach to this.